layers/LAYERS.org

```diff
@@ -1698,18 +1698,18 @@ Features:
 **** Go
 [[file:+lang/go/README.org][+lang/go/README.org]]
 
-This layer adds extensive support for go to Spacemacs.
+This layer adds extensive support for the [[https://golang.org][Go]] programming language.
```

Comment:
Remove all LAYERS.org changes, because if you look at the top of the file it says:
>THIS FILE IS AUTO GENERATED

(they are written in all caps, I didn't yell 😄)
There's a documentation bot that takes the changes from the layers readme and adds them to LAYERS.org.

Reply:
:man_facepalming: Thanks! (and LOL).